SonarQube.Client.Tests/SonarQubeService_GetTaintVulnerabilitiesRequest.cs

```diff
@@ -20,25 +20,28 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
 using SonarQube.Client.Models;
+using SonarQube.Client.Tests.Infra;
+using static SonarQube.Client.Tests.Infra.MocksHelper;
 
 namespace SonarQube.Client.Tests
 {
     [TestClass]
     public class SonarQubeService_GetTaintVulnerabilitiesRequest : SonarQubeService_TestBase
     {
         [TestMethod]
-        public void GetTaintVulnerabilities_NotConnected()
+        public void GetTaintVulnerabilitiesAsync_NotConnected()
```

Comment:
Miscellaneous: added `Async` to the names of the existing tests, since that is the name of the method being tested.